README.md

```diff
@@ -69,7 +69,11 @@ dbname = "..."
 ```
 
 Then run the following commands:
+
 ```
+# Get node modules for the UI
+npm install
```

Comment:
This is only necessary if you want to run Cypress tests, right?

What about adding a `#### Running Cypress Tests` section  beneath `### Local Development outside of App` which briefly explains that you can run tests using Cypress which exercise OpsCenter and you can run them, e.g.
```
npm install # first time only
yarn cypress run
```

(or whatever the expected incantation is)

Reply:
good point. I put it there since they have a shared package.json, but this is also already covered in the cypress readme. do I need to add it here as well?

Reply:
I didn't realize there was a `cypress/README.md`. Completely agree with you, adding this info here is unnecessary. I don't think we need to do anything...